content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Privacy Policy"
+draft: false
+---
+InnerSource Commons is an independent non-for-profit organisation that creates and shares knowledge about InnerSource for its members. 
+
+This privacy policy will explain how our organisation uses the personal data we collect from you when you use our website and contribute to our projects.
+
+## What Data Do We Collect And Why?
+
+You directly provide InnerSource Commons with most of the data we collect. We collect and process data when you:
+
+- Register to our emailing list: InnerSource Commons collects personal identification information (Name and Email Address) to email you with news, events and articles we think you might like.
```

Comment:
currently only mail

Reply:
When I enter my email address (not knowing that I had been subscribed) I am brought to this page. Maybe we should change it if we only want to go with the email address.

![Screenshot 2021-03-03 at 14 24 01](https://user-images.githubusercontent.com/78749459/109820209-b4e14700-7c2c-11eb-8553-f04d5270f876.png)

Reply:
hmm.  The process for me for random mail:

![image](https://i.postimg.cc/8P3BCTqz/screencast-2021-03-03-17-36-18.gif)

And seems your form is located at Mailchimp servers and only in the case when you already subscribed. Should we declare that too?